docs/marketplace/user/back-office-user-guides/202106.0/catalog/products/product-options/managing-product-options.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Managing product options
```

Comment:
2. Move the document from `.../catalog/products/product-options/managing-product-options.md` to `.../catalog/product-options/managing-product-options.md`. There is no `Products` subsection in the BO.